src/databricks_ai_bridge/utils/annotations.py

```diff
@@ -0,0 +1,65 @@
+import inspect
```

Comment:
Are there tests for these annotations that we should also include + include a backlink to the OSS MLflow code (I'm guessing) that we copied from?

Reply:
There are no tests, but I have added the backlink for ref :) 
good idea